### ISSUE TITLE
Fully Connected Neural Networks with Keras

### ISSUE THREAD

**Issue by zacjones93** [eggheadio/egghead-design#130]
# Primary Tag
Keras

# Secondary Tags
Python, Tensorflow

https://egghead.io/courses/fully-connected-neural-networks-with-keras

In this course, we’ll build a fully connected neural network with Keras. This is the most basic type of neural network you can create, but it is a powerful way to answer questions like “How much should this house cost?”, or to answer classification questions like “What type of flower is this?” or “Will the stock price go up or down tomorrow?”. You don’t need to know a lot of python for this course, but some basic python knowledge will be helpful.

After we create the primary network, we’ll explore different ways we can build and configure the network, and evaluate and test the accuracy of each. Then we’ll save the model and learn how to load it and use it to make predictions in the future. Finally, we’ll expose the model as part of a tiny web application that can be used to make predictions.

**Comment by MaggieAppleton** [comment]
Assigned to freelancer – Aleksander

**Comment by MaggieAppleton** [comment]
Finals: https://www.dropbox.com/sh/ug5jsq7cjbu57ft/AACBFJ3XZUYrFc9opGg2K_dSa?dl=0
